ChangeLog

```diff
@@ -36,6 +36,7 @@ ver. 0.9.2 (2014/XX/XXX) - wanna-be-released
      issue (gh-161)
    * filter.d/postfix-sasl.conf - tweak failregex and add ignoreregex to ignore
      system authentication issues
+   * fail2ban-regex does not read '.local' file of given filter (gh-954)
```

Comment:
here as well -- better to make msg descriptive of what has happened (changed, or fixed), not just state the problem in present tense.